### PR TITLE
Android timepicker always open the current time

### PR DESCRIPTION
**Pull request by francescjimenez**
### Expected behaviour
When edit time, android time picker must show the time based on the time of the data

### Actual behaviour
Android date picker always show the current time

### Solution
Change file datepicker.android.js

### CODE REVIEW

File: .idea/inspectionProfiles/Project_Default.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```
Comment (alvaromb) [inline .idea/inspectionProfiles/Project_Default.xml]: Please remove all the `.idea/` files.

File: lib/templates/bootstrap/datepicker.android.js [hunk 2]
```diff
@@ -58,7 +58,15 @@ function datepicker(locals) {
         background={background}
         onPress={function () {
           if (datePickerMode === 'time') {
-            TimePickerAndroid.open({is24Hour: true})
+
+            const now = new Date();
+            const isDate = locals.value && locals.value instanceof Date;
+            let setTime = {
+                hour: (isDate) ? locals.value.getHours() : now.getHours(),
```
Comment (alvaromb) [inline lib/templates/bootstrap/datepicker.android.js]: Can you format your code to 2 space tabs?

**Comment by alvaromb** [comment]
@gcanti seems good to merge for me.

Also, @francescjimenez would like to help us maintain the library.

**Comment by gcanti** [comment]
@alvaromb :+1:

@francescjimenez could you please squash the commits?

**Comment by francescjimenez** [comment]
nice thanks! ;)

**Comment by NickBefore** [comment]
I'm having this problem guys. At which version this has been fixed?

**Comment by francescjimenez** [comment]
In the last version is solved

**Comment by NickBefore** [comment]
Which is? I'm using 0.6.9.

**Comment by NickBefore** [comment]
Sorry, it was my mistake. Solved now.

**Pull request merged**